alpha/lib/model/LiveEntryServerNode.php

```diff
@@ -128,4 +128,21 @@ private function getLiveEntry()
 		
 		return $liveEntry;
 	}
+	
+	public function validateEntryServerNode()
+	{
+		$liveEntry = entryPeer::retrieveByPK($this->getEntryId());
+		if(!$liveEntry)
+		{
+			KalturaLog::debug("Entry with id [{$this->getEntryId()}] not found, will not validate entryServerNode registered");
```

Comment:
This case is abnornal, liveentryID must be found on LiveEntryServerNode, should throw exception